src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -623,7 +625,7 @@ private void drawHUD(Graphics g, boolean includeTileDetails)
 
   private int lastTurnNum = -1;
   private BufferedImage turnNumImage;
-  private void drawTurnCounter(Graphics g, boolean counterIsRight)
+  private void drawTurnCounter(Graphics g, boolean overlayIsLeft)
```

Comment:
Was the turn counter so in the way that it had to move over with the CO overlay? The CO overlay looks funny now that it's not top-justified. Not the end of the world though, I guess.

Reply:
Yes and no. If you're at the top corner of the map and also the window, there's no way to see what's under it (and your cursor can get under it).
I tried moving it under the CO header(s), but that didn't look quite right either.